options/posix/generic/net-if-stubs.cpp

```diff
@@ -1,12 +1,14 @@
 #include <errno.h>
 #include <net/if.h>
+#include <stdlib.h>
 
 #include <bits/ensure.h>
+#include <mlibc/debug.hpp>
 #include <mlibc/posix-sysdeps.hpp>
 
-void if_freenameindex(struct if_nameindex *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+void if_freenameindex(struct if_nameindex *idx) {
+	(void) idx;
```

Comment:
nitpick, but: you can leave the argument unnamed to avoid having to cast it to void